Bucketeer/Sources/Internal/Evaluation/EvaluationInteractor.swift

```diff
@@ -21,85 +21,114 @@ extension EvaluationInteractor {
+                    logger?.debug(message: "Nothing to sync")
+                    completion?(result)
+                    return
+                }
+                let newEvaluations = response.evaluations.evaluations
+                self?.evaluations[user.id] = newEvaluations
+                self?.currentEvaluationsId = newEvaluationsId
+                // https://github.com/bucketeer-io/android-client-sdk/issues/69
+                // evaluatedAt: the last time the user was evaluated. The server will return in the get_evaluations response (UserEvaluations.CreatedAt), and it must be saved in the client
+                // userAttributesUpdated: when the user attributes change via the customAttributes interface, the userAttributesUpdated field must be set to true in the next request.
+                self?.stateStorage.setEvaluatedAt(value: response.evaluations.createdAt)
+                self?.stateStorage.setUserAttributesUpdated(value: false)
+
+                var shouldNotify = true
+                do {
+                    // forceUpdate: a boolean that tells the SDK to delete all the current data and save the latest evaluations from the response
+                    if response.evaluations.forceUpdate {
+                        try self?.forceUpdate(userId: user.id, evaluations: response.evaluations.evaluations)
+                    } else {
+                        shouldNotify = try self?.update(
+                            userId: user.id,
+                            evaluations: response.evaluations.evaluations,
+                            archivedFeatureIds: response.evaluations.archivedFeatureIds) ?? false
                     }
-                    self?.currentEvaluationsId = newEvaluationsId
-                    self?.evaluations[user.id] = newEvaluations
+                } catch let error {
+                    logger?.error(error)
+                    completion?(.failure(error: .init(error: error), featureTag: featureTag))
+                    return
+                }
 
+                if shouldNotify {
                     // Update listeners should be called on the main thread
                     // to avoid unintentional lock on Interactor's execution thread.
                     DispatchQueue.main.async {
                         self?.updateListeners.forEach({ _, listener in
                             listener.onUpdate()
                         })
                     }
-
-                    completion?(result)
-                case .failure:
-                    completion?(result)
                 }
+
+                completion?(result)
+            case .failure:
+                completion?(result)
+            }
         }
     }
 
     func refreshCache(userId: String) throws {
         evaluations[userId] = try evaluationDao.get(userId: userId)
     }
 
-    func clearCurrentEvaluationsId() {
-        currentEvaluationsId = ""
+    func onUpdateUserAttributes() {
```

Comment:
Please rename it to `setUserAttributesUpdated`.

Reply:
because the underlying func for this already is `setUserAttributesUpdated` 
So I made the method name more understandable on when it should be used. 

`onUpdateUserAttributes` ~ indicates this method should be called when user attributes are updated. It follows naming conversations for iOS delegate pattern func